sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringApp.java

```diff
@@ -66,6 +66,12 @@
     /** @return the blob url to upload deployment */
     ResourceUploadDefinition getResourceUploadUrl();
 
+    /**
+     * (Enterprise Tier Only)
+     * @return whether this app has binding to the default Configuration Service
+     */
+    boolean hasConfigurationServiceBinding();
```

Comment:
What does user get from this API? What does it mean for user, what user can do if true, or false?

Reply:
To use the configuration service, app has to be bound to it. This can be done by `withConfigurationServiceBinding()` when defining their apps. 
This API tells the user whether this app has a binding to the configuration service. If not, they can use `withConfigurationServiceBinding()` in their app update.

From their document:
> To use the centralized configurations, you must bind the app to Application Configuration Service for Tanzu. After binding the app, you'll need to configure which pattern to be used by the app

Reply:
I'll add this to javadoc.

Reply:
So binding is a must for enterprise tier? Does it mean you should instead make the binding step in create mandatory instead of optional (if you have this step).

Reply:
It's not mandatory as far as I know. When user choose not to use configuration service, they can skip binding their app to it and it's allowed.
> Or it is not required, and even if not configured, service will still make a default one?

If not configured, no default binding exists.
I considered binding the app for the user if not configured, but that differs from the behavior of either portal or cli(they require explicit app binding declaration). So I stayed that way.